### ISSUE TITLE
Login

### ISSUE THREAD

**Issue by dri94** [kaloyan-raev/hello-storj#35]
Before the v3 change you could login to storj with a Key like`Keys(username, password, mnemonic)`, however now that has changed. How do we get the apikey and encryption access to create a key now? are they user specified?

**Comment by kaloyan-raev** [comment]
Yes, this concept change in V3. You now need an "access grant" for accessing data on the Storj network.

The access grant consist of the parts:
- Satellite address
- API Key
- Encryption Access

The API key is created by the user in the web interface on the satellite: https://documentation.tardigrade.io/getting-started/uploading-your-first-object/create-an-api-key

The Encryption Access can be generated from a human-readable passphrase.

Here is an example in the Android library that can be used to create an access grant from the API key and Passphrase: https://github.com/storj/uplink-android#creating-new-scope-from-passphrase

Note that in the Android library the "access grant" is still called "scope". This is the old name. We will soon update the Android library to reflect that.

**Comment by dri94** [comment]
@kaloyan-raev is there any way to expose a list of satellites with their address? So we can have the user select one. Also should we be using android-libstorj or uplink-android? The former seems to have better interfaces for handling login, etc.
